src/phaser/scene/VillageScene/index.js

```diff
@@ -21,15 +24,11 @@ class VillageScene extends Phaser.Scene {
   create() {
     const socket = io('http://localhost:3001/');
 
-    const villageMap = MapManager.createVillageMap(new DefaultMapBuilder(this));
+    const villageMap = MapManager.createMap(this, 'defaultBackground', 'villageMap', 'tileSet');
```

Comment:
assets load는 preload에서 하기 때문에 assets에 대한 key는 주입받도록 구현.

Reply:
```suggestion
    const DEFAULT_BACKGROUND_KEY = 'defaultBackgroud';
    const DEFAULT_GROUND_KEY = 'defaultGround';
    const VILLAGE_MAP_KEY = 'villageMap';
    const TILE_SET_KEY = 'tileSet';
    
    const villageMap = MapManager.createMap(this, DEFAULT_BACKGROUND_KEY, VILLAGE_MAP_KEY, TILE_SET_KEY);
```

defaultBackround, ground, villageMap, tileSet 이 this.load 에서도 그렇고 createMap에서도 그렇고 주입받아질 일이 많아질 것 같은데 문자 리터럴 보다 상수화 시키는게 어떨까요?